_docs/layout/architecture.md

```diff
@@ -30,8 +30,8 @@ Application.
 The global layer wraps around the entire Helix-UI and at least some portion of
 it is always visible, regardless of what the user is currently viewing. The
 global layer is best thought about as a collection of offscreen elements that
-will remain hidden until some action is taken by the user. Once an action is
-taken the global layer services up contextually sensitive info and workflows
+remain hidden until some action is taken by the user. After an action is
+taken, the global layer serves contextually sensitive info and workflows
 for managing platform(global) services. View the
```

Comment:
I'd add a space before the `(` here. Mind grabbing that for us as you're in here?